pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -215,6 +218,489 @@ def get_strategic_model_unit_container():
 )
 
 
+# TODO: move this to utility
+class MissingDataError(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+
+
+# TODO: Move this function to a utility
```

Comment:
I agree with moving this to a utility. However, why define a function to return a constant list? Why not just assign the list to a variable instead?

Reply:
I like the function because it feels cleaner when calling from different files. The intention is for this list to be defined only once throughout PARETO and accessed in other files with the get_..() function

FYI  commit is coming, but quite a lot is changing since this versions, so replying to these comments